packages/lib/src/components/Card/components/CardInput/components/StoredCardFields.tsx

```diff
@@ -41,6 +41,13 @@ export default function StoredCardFields({
                     >
                         {expiryMonth} / {expiryYear}
                     </div>
+                    <div className="adyen-checkout__field__exp-date_hint_wrapper">
```

Comment:
Do we need the date icon in the Stored Card?

Reply:
It isn't, removed it!